roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -764,4 +764,48 @@ class TransitionSpec extends FlatSpec with Matchers {
     val end = VirtualMachine.executeSeq(codevec, start)
     end.ctxt.ctxt.rslt shouldBe Fixnum(7)
   }
+
+  "OpXmitArgXtnd" should "save the result of xmit function into the specified argument for future use, the argument is specified by the following byte" in {
+
+    /**
+      * This opcode sequence will never be generated by the RBL
+      * compiler.
+      *
+      * One of the expressions that use the OpXmitArgXtnd shows below (there
+      * should be more than 16 times '(+1 1)'), which compiled into about 124 opcodes.
+      *
+      * (+ (+ 1 1) (+ 1 1) (+ 1 1) (+ 1 1)
+      *    (+ 1 1) (+ 1 1) (+ 1 1) (+ 1 1)
+      *    (+ 1 1) (+ 1 1) (+ 1 1) (+ 1 1)
+      *    (+ 1 1) (+ 1 1) (+ 1 1) (+ 1 1)
+      *    (+ 1 1))
+      *
+      *
+      * litvec:
+      *   0:	{RequestExpr}
+      * codevec:
+      *   0:	alloc 17
+      *   1:  push/alloc 2
+      *   2:	lit 1,arg[0]
+      *   3:	lit 1,arg[1]
+      *   4:	xfer global[+],trgt
+      *   6:	xmit 2,arg[16]
```

Comment:
It's not clear here that the real output is much longer. Please insert the `codevec` up to index 19 and then indicate that it continues from there (`[...]`) and after that add the `codevec` from index 117 to the end.